File: Day four/python/After Break/task_Eight.py








total_sum =0
average =0
counter =0
    
    
for number in range(1,11):

    scores= int(input("Enter scores: "))
    if (scores <0 and scores >100) :
        print("Invalid score")
    else :
        total_sum += scores;

print ("The sum is: ",total_sum)


 

File: Day four/java/After break/TaskEight.java







import java.util.Scanner;
public class TaskEight {
    public static void main (String [] args) {
    Scanner inputcollector = new Scanner (System.in);

    int sum =0;
    int counter =0;
    
    System.out.print("Enter ten score: ");    
for (int count =1; count <=10; count++) {
    int scores = inputcollector.nextInt();
    if (scores <0 ) {
    System.out.println ("Invalid score");
    }
    else {
        sum += scores;
    }
}
        System.out.println ("The total sum is: "+sum);
    }   
}
